Tilers/CityTiler/database_accesses.py

```diff
@@ -1,23 +1,36 @@
+"""
+Notes on the 3DCityDB database general structure
```

Comment:
Dispatch those (really usefull) design comments concerning the CityGML conceptual model to the respective new locations of the sql queries (that is in the citim_relief.py an citim_building.py).